src/diffuse/main.py

```diff
@@ -960,6 +960,7 @@ def create_submenu(self, sections):
                 else:
                     if data is not None:
                         data = GLib.Variant.new_string(data)
+                    accel = accel.replace("_", "-")
```

Comment:
This is not working as the `Resources` class still use "old" syntax (see `Resources.__init__`). I think the proper solution would be to refactor this too. I'm not fond of code that fixes strings on the fly, especially if we have control over these strings :wink: 

I'll work on a fix for that

Reply:
I though of that but decided it would be way to many changes (docs, translations etc) and it could break existing user's configuration files. So I just made this hack.
If it's OK to rename all action ids, than it would be cleaner solution.

Reply:
I see your point.

Right now, I think the code base is not in a very good shape and has a lot of tech debts. So if it's to modernize the code base, I'm ok with some refactoring.

I can see that this PR is becoming quite big, so we'll try and wrap it up, but if you want to continue modernizing the codebin firther PRs, you are very welcome to do so! (Maybe we'll eventually migrate to GTK4? Or even have unit tests?)

Anyway, I'm advancing on the menus/actions/accelerators refactoring. It's a bit of a tedious process, but I'm learning a lot about GTK menus and actions. I'm quite happy with the results right now (hoping I'll not hit a wall 🤞)

Reply:
I'll probably try a more quiet approach and go with what you made right now and incrementally make changes towards a cleaner source code.

I've got one question though: why replacing `_` with `-`? I don't see it mentioned in the GTK documentation that the action should not have `_` in their name.

Reply:
Ok, I see:

> GLib-GIO-CRITICAL **: 14:19:41.573: g_action_print_detailed_name: assertion 'g_action_name_is_valid (action_name)' failed